### DISCUSSION TITLE
Selecting a Widget Style with NodeJS

### DISCUSSION THREAD

**Discussion by matheusdsg27** [slint-ui/slint#4540] [General]
Hi guys, in the slint simulator integrated with visual studio I saw that there are several pre-defined styles, how can I apply these styles to my application?

I'm using NodeJS and npm start. (reference: https://slint.dev/releases/1.4.1/docs/slint/src/advanced/style#selecting-a-widget-style)

{
    "name": "counter",
    "version": "1.0.0",
    "main": "src/main.js",
    "type": "module",
    "dependencies": {
        "slint-ui": "^1.4.1"
    },
    "scripts": {
        "start": "node ."
    }
}

**Comment by ogoffart** [comment]
That page should be updated to include s link to https://slint.dev/releases/1.4.1/docs/node/interfaces/LoadFileOptions#style
Which you can pass as an option to loadFile.


Also the SLINT_STYLE env variable should work
